dev-server/ui-playground/shared/components/PlaygroundApp.js

```diff
@@ -50,10 +65,11 @@ export default function PlaygroundApp() {
     <main className="PlaygroundApp">
       <div className="PlaygroundApp__sidebar">
         <div className="PlaygroundApp__sidebar-home">
-          <a href={baseUrl} onClick={e => navigate(e, '/')}>
+          <a href={baseURL} onClick={e => navigate(e, '/')}>
             <SvgIcon name="logo" />
           </a>
         </div>
+        <h2>Common Patterns</h2>
```

Comment:
The left border width of the heading is slightly different than the hover state for buttons, which looks a little out of place:

<img width="335" alt="Sidebar border width" src="https://user-images.githubusercontent.com/2458/114158886-1cba3a00-991d-11eb-9eee-01fa71f45d1e.png">

Reply:
Removing this left-border here.